docs/content/backup_restore.md

```diff
@@ -36,7 +36,7 @@ You can specify the behavior of the command with some options detailed below.
 
 <div class="standalone">
 
-By default, the command will backup the database from the default connection.
+By default, the command will back up the database from the default connection.
```

Comment:
Same.

Reply:
Not there. This is the verb we are using here. I believe the contraction is only allowed for the noun.